docs/translations/translations.json

```
@@ -2,6 +2,7 @@
   "adblock": "If you don't mind-tech related ads (no tracking or remarketing), and want to keep us running, please whitelist Material-UI in your blocker.",
   "api-docs": {
     "componentName": "Component name",
+    "cssComponent": "As a CSS utility the {{name}} component also supports all <a href=\"/system/properties\"><code>system</code></a> properties. You can use them as prop directly on the component.",
```

To avoid a redirection 
```suggestion
    "cssComponent": "As a CSS utility the {{name}} component also supports all <a href=\"/system/properties\/"><code>system</code></a> properties. You can use them as prop directly on the component.",
```